webpack/assets/javascripts/compute_resource/openstack.js

```diff
@@ -23,7 +24,7 @@ export function schedulerHintFilterSelected(item) {
         // eslint-disable-next-line function-paren-newline
         $('#scheduler_hint_wrapper').html(
           // eslint-disable-next-line no-undef
-          Jed.sprintf(__('Error loading scheduler hint filters information: %s'), error));
+          jed.sprintf(__('Error loading scheduler hint filters information: %s'), error));
```

Comment:
`sprintf` is used quite often. Would it make sense to export it from `I18n` directly? We could then use shorter:
```js
import { sprintf, translate as __ } from '../react_app/common/I18n';
// ...
sprintf(__('Error loading scheduler hint filters information: %s'), error));
```